examples/java/buildSrc/build.gradle

```
@@ -25,5 +25,7 @@ ext {
 }
 
 dependencies {
+  implementation "com.diffplug.spotless:spotless-plugin-gradle:6.2.2"
   implementation "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
+  implementation 'org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:3.3'
```

just nitpicks - alphabetical order and consistency with quotes

```suggestion
  implementation "com.diffplug.spotless:spotless-plugin-gradle:6.2.2"
  implementation "org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:3.3"
  implementation "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
```

---

Good spot, thanks!